setup.py

```diff
@@ -25,9 +25,9 @@
         'ordereddict',
```

Comment:
I think you can remove ordereddict - I believe it's been added to python 3 - see https://docs.python.org/3/library/collections.html#collections.OrderedDict

Reply:
Thanks!